### ISSUE TITLE
NotFoundException

### ISSUE THREAD

**Issue by jakobroehrl** [matiasdelellis/facerecognition#574]
Hi, 
could you help me with that?
Tested with NC24, but I'm not sure if it's a NC24 problem or it occurs before.

```
oot@c:~# sudo -u www-data /usr/bin/php /var/www/nextcloud/occ face:background_job -t 900
1/10 - Executing task CheckRequirementsTask (Check all requirements)
2/10 - Executing task CheckCronTask (Check that service is started from either cron or from command)
3/10 - Executing task LockTask (Acquire lock so that only one background task can run)
4/10 - Executing task DisabledUserRemovalTask (Purge all the information of a user when disable the analysis.)
5/10 - Executing task StaleImagesRemovalTask (Crawl for stale images (either missing in filesystem or under .nomedia) and remove them from DB)
Error during background task execution
If error is not transient, this means that core component of face recognition is not working properly
and that quantity and quality of detected faces and person will be low or suboptimal.
You probably want to file an issue (please include exception below) to: https://github.com/matiasdelellis/facerecognition/issues

In Root.php line 346:

  [OCP\Files\NotFoundException]

```

**Comment by matiasdelellis** [comment]
Hi @jakobroehrl 
Please test last release..

Thanks.

**Comment by jakobroehrl** [comment]
fixed with 0.9.5
Thanks a lot!
